cs/src/attributes/Attributes.cs

```diff
@@ -142,8 +142,14 @@ public sealed class RequiredOptionalAttribute : Attribute
     /// <summary>
     /// Specifies user defined schema attribute
     /// </summary>
-    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Enum | 
-        AttributeTargets.Interface | AttributeTargets.Field | AttributeTargets.Property, 
+    [AttributeUsage(
+        AttributeTargets.Class |
+        AttributeTargets.Enum |
+        AttributeTargets.Field |
+        AttributeTargets.Interface |
+        AttributeTargets.Method |
```

Comment:
I'm guessing this was the missing one that caused the reported failure right?

Reply:
Yup.